crates/shielded_token/src/masp/shielded_wallet.rs

```diff
@@ -255,7 +302,14 @@ impl<U: ShieldedUtils + MaybeSend + MaybeSync> ShieldedWallet<U> {
         &mut self,
         transaction: &Transaction,
         update_witness_map: bool,
-    ) {
+        #[cfg(feature = "historic")] update_history: Option<IndexedTx>,
+    ) -> Result<(), eyre::Error> {
+        #[cfg(feature = "historic")]
+        let used_conversions = transaction.sapling_bundle().map_or_else(
+            || false,
```

Comment:
```suggestion
        let used_conversions = transaction.sapling_bundle().map_or(
            false,
```

Reply:
👍🏻 Done in 52ef12d8f7d621ff65375e72e8d76138085cc063